MediaGalleryUi/view/adminhtml/web/js/image-uploader-standalone-page.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'Magento_MediaGalleryUi/js/image-uploader',
+    'underscore'
+], function (ImageUploader, _) {
+    'use strict';
+
+    return ImageUploader.extend({
+        defaults: {
+            // eslint-disable-next-line max-len
+            actionsPath: 'standalone_media_gallery_listing.standalone_media_gallery_listing.media_gallery_columns.thumbnail_url_actions',
```

Comment:
I'd pass the custom setting to the component from the backend instead of creating a new component

Reply:
Yes but in uicomponent it will be merged and replaced from original listing

Reply:
The original uploader options (that are defaults) should be replaced by standalone uploader options (provided as configuration for the component from backend)

Reply:
ok